funnel/templates/index.html.jinja2

```diff
@@ -485,14 +485,13 @@
 {% endblock %}
 
 {% block footerscripts %}
+  {% assets "js_jquerysuccinct" -%}
+    <script type="text/javascript" src="{{ ASSET_URL }}"></script>
+  {%- endassets -%}
+  <script src="{{ url_for('static', filename=asset_path('home')) }}" type="text/javascript"></script>
```

Comment:
What is this asset? Filename seems deformed.

Reply:
Changed it index.js